cmd/admin/cert.go

```diff
@@ -221,6 +224,23 @@ func (a *admin) serialsFromRegID(ctx context.Context, regID int64) ([]string, er
 	return serials, nil
 }
 
+func (a *admin) checkValidSerial(serial string) (string, error) {
+	serialStrip := func(r rune) rune {
+		switch {
+		case unicode.IsLetter(r):
+			return r
+		case unicode.IsNumber(r):
```

Comment:
We probably want `IsDigit` here: `IsNumber` is true for things like fractions and roman numerals.
```suggestion
		case unicode.IsDigit(r):
```

That probably doesn't matter too much, as core.ValidSerial will reject anything that's not hex, though.  Nobody's going to have a serial number deliminated with unicode fractions